sprocs/index.js

```diff
@@ -175,6 +175,7 @@ module.exports = {
             'workspace_hosts_recapture_draining.sql',
             'workspace_hosts_drain_extra.sql',
             'workspace_hosts_find_terminable.sql',
+            'select_log.sql',
```

Comment:
Can we call this `assessment_instances_select_log`? The PL naming convention for sprocs is to start with the table name they are most closely associated with.

Reply:
And in `sprocs/index.js` list it with the other `assessment_instances` sprocs.